sdk/communication/communication-call-automation/review/communication-call-automation.api.md

```diff
@@ -213,7 +213,7 @@ export class CallMedia {
     constructor(callConnectionId: string, endpoint: string, credential: KeyCredential | TokenCredential, eventProcessor: CallAutomationEventProcessor, options?: CallAutomationApiClientOptionalParams);
     cancelAllOperations(): Promise<CancelAllMediaOperationsResult>;
     play(playSources: (FileSource | TextSource | SsmlSource)[], playTo: CommunicationIdentifier[], options?: PlayOptions): Promise<PlayResult>;
-    playToAll(playSources: (FileSource | TextSource | SsmlSource)[], options?: PlayOptions): Promise<PlayResult>;
+    playToAll(playSources: (FileSource | TextSource | SsmlSource)[], options?: PlayOptions, interruptCallMediaOperation?: boolean): Promise<PlayResult>;
```

Comment:
Unfortunately, the standard for JS is that all the optional parameters have to fall into the Options object.

We may probably need to split the PlayOptions object into PlayOptions and PlayToAllOptions, the same way we do in other languages.

But that is going to be a breaking change. Please sync with Min and Paresh to verify options.